test/typings/test-d/select.test-d.ts

```diff
@@ -115,6 +115,11 @@ async function testSelectSingle(db: Kysely<Database>) {
     .$narrowType<NarrowTarget>()
     .execute()
 
+  expectType<
+    | { callback_url: string; queue_id: null }
+    | { callback_url: null; queue_id: string }
+  >(r15)
```

Comment:
`r15` was never tested at all before. so this was just a missing test; its result is not changed by this pr